tests/gplates_1_3/screen-output

```diff
@@ -1,13 +1,6 @@
------------------------------------------------------------------------------
--- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
---     . version 1.2.pre
---     . running in DEBUG mode
---     . running with 1 MPI process
---     . using Trilinos
------------------------------------------------------------------------------
 
 Number of active cells: 768 (on 4 levels)
-Number of degrees of freedom: 10,656 (6,528+864+3,264)
+Number of degrees of freedom: 10.656 (6.528+864+3.264)
```

Comment:
This is a trick one. Your shell outputs numbers in the german numbering convention. I can simply overwrite the output with results from my machine after all other changes are done.